test/test_fields.py

```diff
@@ -263,11 +309,13 @@ class CallableEmptyValue(MongoModel):
     def test_mutable_default(self):
         class Article(MongoModel):
             tags = fields.ListField(fields.CharField())
+            dict = fields.DictField()
```

Comment:
Suggest naming `dct` or `extra` instead of `dict`. Also is this field used anywhere?